Gemfile

```diff
@@ -7,7 +7,7 @@ gem 'rspec-expectations'
 gem 'rspec-mocks'
 gem 'chef'
 gem 'foodcritic'
-gem 'rubocop'
+gem 'cookstyle'
```

Comment:
Cool! I haven't kept up with Chef lately, good to see tooling more specific to Chef here.